yarn/pom.xml

```diff
@@ -109,6 +109,16 @@
       <properties>
         <jersey.version>1.9</jersey.version>
       </properties>
+      <!-- Enable hadoop-2.2 profile when the Hadoop version is 2.2.0, which is now 
+      the default. Those building against 2.2.x or some distro-specific variation of 
+      that string would still have to manually enable -Phadoop-2.2.
+      -->
+      <activation>
+       <property>
+        <name>hadoop.version</name>
```

Comment:
This gets kinda circular since the profile also sets this version. I think we've had profiles set versions instead of the other way around. This profile is in theory not just for "2.2.0"